### PR TITLE
New definitions of "H", "L" and "M"

### PR DESCRIPTION
**Pull request by Clayblockunova**

### CODE REVIEW

**Comment by fasttime** [comment]
Thanks! I think we can also add a similar definition for `"H"`. This would be advantageous in some conditions, for example when only feature `ANY_DOCUMENT` is available.

**Comment by Clayblockunova** [comment]
Sorry. Since availability of `ANY_DOCUMENT` overlaps with `HISTORY` in major platforms, idk is there any possibility of which  `ANY_DOCUMENT` is avaliable but `HISTORY` is unavaliable.

**Comment by fasttime** [comment]
That's not a problem. JScrewIt already includes many definitions that will never be used on any real platform, for "D" for 
example: https://github.com/fasttime/JScrewIt/blob/83cb9fb8e9f7040c42bf19fb9a54a352bf14ab1b/src/lib/definitions.js#L805-L806
Definitions are cheap, so it's okay to have any number of definitions, as long as they produce the shortest code in a particular combination of features. By contrast, features are expensive, so new features are only added when they are useful for one of the supported engines, or a combination of engines.

**Comment by Clayblockunova** [comment]
Definition of "H" added.

File: src/lib/definitions.js [hunk 1]
```diff
@@ -821,6 +821,7 @@ function getFHPaddingEntries(index)
         ],
         'H':
         [
+            define('(RP_2_WS + document.createElement(false))[10]', ANY_DOCUMENT),
```
Comment (fasttime) [inline src/lib/definitions.js:824]: It's better to use `(RP_3_WA + ...)[11]` (saves 1 character).

**Comment by fasttime** [comment]
LGTM, thanks @Clayblockunova! Your help is very appreciated.

**Pull request merged**